--- vulcan-0.5.1/render.c~	2007-10-15 03:21:55.000000000 +0200
+++ vulcan-0.5.1/render.c	2007-10-29 10:55:44.000000000 +0100
@@ -625,9 +625,9 @@
 	int i;
 
 	static const char *mesh_names[NUM_PIECES] = {
-		"data/models/pawn-low.wrl", "data/models/rook-low.wrl",
-		"data/models/knight-low.wrl", "data/models/bishop-low.wrl",
-		"data/models/queen-low.wrl", "data/models/king-low.wrl" };
+		"/usr/share/vulcan/models/pawn-low.wrl", "data/models/rook-low.wrl",
+		"/usr/share/vulcan/models/knight-low.wrl", "data/models/bishop-low.wrl",
+		"/usr/share/vulcan/models/queen-low.wrl", "data/models/king-low.wrl" };
 
 	for (i = 0; i < NUM_PIECES; i++) {
 		load_piece(&piece_models[i], mesh_names[i]);
