--- vulcan-0.9/Makefile~	2007-12-01 18:08:20.000000000 +0100
+++ vulcan-0.9/Makefile	2007-12-01 18:08:19.000000000 +0100
@@ -1,6 +1,6 @@
 VERSION	= 0.9
 
-PREFIX = /usr/local
+PREFIX = /usr
 BIN = $(PREFIX)/bin
 DATA_DIR = $(PREFIX)/share/vulcan
 
@@ -13,7 +13,7 @@
 LD	= gcc
 YACC	= bison -y
 LEX	= flex
-CFLAGS	= -O2 -g -Wall \
+CFLAGS	= -Wall \
 	-I/usr/X11R6/include -I/usr/local/include -Ilib3d \
 	-DDATA_DIR=\"$(DATA_DIR)\" -DFONT_DIR=\"$(FONT_DIR)\" \
 	-DMODEL_DIR=\"$(MODEL_DIR)\" -DTEXTURE_DIR=\"$(TEXTURE_DIR)\"
@@ -93,6 +93,7 @@
 
 .c.o:
 	$(CC) $(CFLAGS) -c $<
+	$(CC) $(OPTFLAGS) $(CFLAGS) -c $<
 
 lib3d.o:
 	$(MAKE) -C lib3d
@@ -118,17 +119,15 @@
 	(cd ..; rm vulcan-$(VERSION))
 
 install: $(TARGET) chessmodels
-	install -s -m 755 $(TARGET) $(BIN)
-	install -m 755 -d $(DATA_DIR)
-	install -m 755 -d $(FONT_DIR)
-	cp data/fonts/* $(FONT_DIR)
-	chmod 644 $(FONT_DIR)/*
-	install -m 755 -d $(MODEL_DIR)
-	./chessmodels > $(MODEL_DIR)/chess-models.modeldef
-	chmod 644 $(MODEL_DIR)/*
-	install -m 755 -d $(TEXTURE_DIR)
-	cp data/textures/* $(TEXTURE_DIR)
-	chmod 644 $(TEXTURE_DIR)/*
+	mkdir -p $(DESTDIR){$(BIN),$(FONT_DIR),$(MODEL_DIR),$(TEXTURE_DIR)}
+	install $(TARGET) $(DESTDIR)$(BIN)
+	install -d $(DESTDIR)$(DATA_DIR)
+	install -d $(DESTDIR)$(FONT_DIR)
+	cp data/fonts/* $(DESTDIR)$(FONT_DIR)
+	install -d $(DESTDIR)$(MODEL_DIR)
+	./chessmodels > $(DESTDIR)$(MODEL_DIR)/chess-models.modeldef
+	install -d $(DESTDIR)$(TEXTURE_DIR)
+	cp data/textures/* $(DESTDIR)$(TEXTURE_DIR)
 
 distclean:
 	rm -f $(BIN)/$(TARGET)
--- vulcan-0.9/lib3d/Makefile~	2007-12-01 18:12:53.000000000 +0100
+++ vulcan-0.9/lib3d/Makefile	2007-12-01 18:12:38.000000000 +0100
@@ -1,7 +1,7 @@
 CC	= gcc
 MV	= mv
 LD	= gcc
-CFLAGS	= -O2 -g -Wall
+CFLAGS	= -Wall
 YFLAGS	= -d
 LFLAGS	= 
 YFILES	= vrml.y modeldef.y fontdef.y
@@ -29,7 +29,7 @@
 	ar cr $@ $(OBJS)
 
 .c.o:
-	$(CC) $(CFLAGS) -c $<
+	$(CC) $(OPTFLAGS) $(CFLAGS) -c $<
 
 clean:
 	rm -f $(OBJS) $(TARGET)
