--- vulcan-0.4/vulcan-ui.c~	2007-10-03 21:24:12.000000000 +0200
+++ vulcan-0.4/vulcan-ui.c	2007-10-09 19:24:17.000000000 +0200
@@ -239,9 +239,9 @@
 	int i;
 
 	static const char *mesh_names[NUM_PIECES] = {
-		"data/models/pawn-low.wrl", "data/models/rook-low.wrl",
-		"data/models/knight-low.wrl", "data/models/bishop-low.wrl",
-		"data/models/queen-low.wrl", "data/models/king-low.wrl" };
+		"data/models/pawn-low.wrl", "/usr/share/vulcan/models/rook-low.wrl",
+		"data/models/knight-low.wrl", "/usr/share/vulcan/models/bishop-low.wrl",
+		"data/models/queen-low.wrl", "/usr/share/vulcan/models/king-low.wrl" };
 
 	for (i = 0; i < NUM_PIECES; i++) {
 		load_piece(&piece_models[i], mesh_names[i]);
